ofl/bonanova/METADATA.pb

```diff
@@ -0,0 +1,40 @@
+name: "Bona Nova"
+designer: "Capitalics, Mateusz Machalski, Andrzej Heidrich"
+license: "OFL"
+category: "SERIF"
+date_added: "2021-04-06"
+fonts {
+  name: "Bona Nova"
+  style: "normal"
+  weight: 400
+  filename: "BonaNova-Regular.ttf"
+  post_script_name: "BonaNova-Regular"
+  full_name: "Bona Nova Regular"
+  copyright: "Copyright 2017 The Bona Nova Project Authors (https://github.com/kosmynkab/Bona-Nova)"
+}
+fonts {
+  name: "Bona Nova"
+  style: "italic"
+  weight: 400
+  filename: "BonaNova-Italic.ttf"
+  post_script_name: "BonaNova-Italic"
+  full_name: "Bona Nova Italic"
+  copyright: "Copyright 2017 The Bona Nova Project Authors (https://github.com/kosmynkab/Bona-Nova)"
+}
+fonts {
+  name: "Bona Nova"
+  style: "normal"
```

Comment:
Why ? Unless I forgot something, the font has all the necessary bits and style conventions… is this a packager issue?